packages/material-ui-lab/src/Autocomplete/Autocomplete.js

```diff
@@ -340,6 +341,14 @@ const Autocomplete = React.forwardRef(function Autocomplete(props, ref) {
     }
   }
 
+  if (startAdornment) {
+    const more = startAdornment.length - filterMax;
```

Comment:
Users can render whatever they wish with `renderTags`, we have no guarantee that `startAdornment` is  going to be an array. I think that we should assert it first

Reply:
I see, I've add a check to it. Can you verify please?